src/notificationhandler.h

```diff
@@ -58,7 +62,7 @@ class NotificationHandler : public QObject {
 
  private:
   virtual void notifyInternal(Message type, const QString& title,
-                              const QString& message, int timerMsec);
+                              const QString& message, int timerMsec = 5);
```

Comment:
Why this default value? Move it to constants.h

Reply:
I was using `0`, but that will throw and error on MacOS. So I swtiched to 5... Could have been 1. Anyways, I'll move it and add a comment.